### DISCUSSION TITLE
表格获取的html渲染样式丢失

### DISCUSSION THREAD

**Discussion by haipengsong** [wangeditor-next/wangEditor-next#490] [General]
## 问题描述
作者您好：
在富文本中使用表格获取html进行预览渲染，发现表格样式丢失，查阅原官方文档解释是需要自定义样式：
<img width="978" alt="image" src="https://github.com/user-attachments/assets/26d12f6c-88aa-4d29-aa5d-c716b232fac7" />
<img width="841" alt="image" src="https://github.com/user-attachments/assets/b6d9a85d-1cc1-41aa-bd46-e6959cb72681" />
但是现在我们的使用场景是富文本编辑的内容是用于发邮件的，所以在项目里预览可以通过自定义样式去解决，但是发出去的邮件就没办法修改样式了，这块有什么好的解决办法吗，而且现在只发现了表格，不知道还有没有其他的元素需要自定义样式。

富文本编辑效果如下：
<img width="775" alt="image" src="https://github.com/user-attachments/assets/a202fd06-5906-470d-8d86-58ec3dd77543" />

不自定义样式的预览效果如下：
<img width="619" alt="image" src="https://github.com/user-attachments/assets/84b47747-65ec-4ec8-b527-8e6bd3e06627" />

## wangEditor 版本

5.6.15

## 是否查阅了文档 ？

（文档链接 [https://cycleccc.github.io/docs/](https://cycleccc.github.io/docs/) ）

*是*

## 最小成本的复现步骤

（请告诉我们，如何**最快的**复现该问题？）

- 步骤一
- 步骤二
- 步骤三

**Comment by cycleccc** [comment]
`发出去的邮件就没办法修改样式了`   指的是发出去的邮件再注入富文本编辑器中无法修改吗？

**Comment by haipengsong** [comment]
> `发出去的邮件就没办法修改样式了` 指的是发出去的邮件再注入富文本编辑器中无法修改吗？

不是，是在配置邮件内容时使用了富文本编辑器，需要把编辑器生成的html最后当作邮件发送出去，保留富文本内编辑的样式，所以编辑器生成的html没有内联样式的话就会出现样式丢失的问题

**Comment by cycleccc** [comment]
`保留富文本内编辑的样式` 可以通过将自定义样式注入纯 HTML 中实现，这个不需要修改底层。
wangeditor 的样式也有导出对应的文件，这个可以看文档。
